### PR TITLE
Changelog for 1.4.0 release

### PR DESCRIPTION
**Pull request by sueann**
Changelog for the 1.4.0 release - auto-generated then manually edited for language.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,5 +1,29 @@
...
+- Add functions ``get_run``, ``get_experiment``, ``get_experiment_by_name`` to the fluent API (#1923, @fhoering)
+- Use Plotly as artifact image viewer to make it interactive (#1934, @harupy)
+- Enable scrolling to zoom in metric and run comparison plots (#1929, @harupy)
+- Added support of viewfs URIs for HDFS federation for artifacts (#1947, @t-henri)
+
+Bug fixes and documentation updates:
+
+- Make detection of ``MLproject`` files case-insensitive (#1981, @smurching)
+- Fix encoding of metric names in MLflow UI URLs (#1968, @smurching)
+- Get ``models serve`` CLI command to work in Windows (#1949, @rboyes)
```
Comment (stbof) [inline CHANGELOG.rst]: Get ``models serve`` CLI command to work in Windows  -> ``models serve`` CLI command now works in Windows

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,5 +1,29 @@
...
+- Fix dependency-installation bug in Java MLflow model scoring server (#1913, @smurching)
+
+Small bug fixes and doc updates (#1914, @jerrygb; #1978, #1894, @smurching; #1940, #1900, #1904, @mparkhe; #1966, #1955, #1952, @Djailla; #1932, #1935, @harupy)
```
Comment (stbof) [inline CHANGELOG.rst]: Fix dependency-installation bug -> Fix dependency installation bug

File: CHANGELOG.rst [hunk 3]
```diff
@@ -1,5 +1,29 @@
 Changelog
 =========
+1.4.0 (2019-10-30)
+-----------------------
+MLflow 1.4.0 includes several major features:
+
+- Model Registry. MLflow 1.4 adds an experimental model registry feature, where you can manage, version, and keep lineage of your production models. (#1943, @mparkhe, @Zangr, @sueann, @dbczumar, @smurching, @gioa, @clemens-db, @pogil, @mateiz)
+- Keras support for TensorFlow 2.0. MLflow Keras model saving, loading, and logging has been updated to be compatible with TensorFlow 2.0.  (#1927, @juntai-zheng)
+- Autologging for TensorFlow 2.0. Autologging for ``tf.estimator`` and ``tf.keras`` models has been updated to be compatible with TensorFlow 2.0. The same functionalities of autologging in TF 1.x will be offered in TF 2.0, namely when fitting tf.keras models and when exporting saved tf.estimator models. (#1910, @juntai-zheng)
+
```
Comment (stbof) [inline CHANGELOG.rst]: Suggest a category - TensorFlow 2.0 updates and put these two under it.

File: CHANGELOG.rst [hunk 4]
```diff
@@ -1,5 +1,29 @@
...
+More features and improvements:
+
+- Add functions ``get_run``, ``get_experiment``, ``get_experiment_by_name`` to the fluent API (#1923, @fhoering)
+- Use Plotly as artifact image viewer to make it interactive (#1934, @harupy)
```
Comment (stbof) [inline CHANGELOG.rst]: Use Plotly as artifact image viewer to make it interactive 

What does "it" appy to?

**Review by ghost — CHANGES_REQUESTED** [review]
See inline comments

File: CHANGELOG.rst [hunk 5]
```diff
@@ -1,5 +1,29 @@
 Changelog
 =========
+1.4.0 (2019-10-30)
+-----------------------
+MLflow 1.4.0 includes several major features:
+
+- Model Registry. MLflow 1.4 adds an experimental model registry feature, where you can manage, version, and keep lineage of your production models. (#1943, @mparkhe, @Zangr, @sueann, @dbczumar, @smurching, @gioa, @clemens-db, @pogil, @mateiz)
```
Comment (mateiz) [inline CHANGELOG.rst]: It might be better to call the registry beta so people know that it's still early on.

**Review by ghost — APPROVED** [review]

**Pull request merged**